### DISCUSSION TITLE
How to force UNIX line endings

### DISCUSSION THREAD

**Discussion by r-sergeant** [zed-industries/zed#18999] [Feature Requests]
Hi community,

I know the Windows version is not production ready yet, but I use it almost daily. It seems it saves files with Windows line endings and I can find no way to convert these to UNIX line endings. This is fairly annoying as most of my development is done on Linux and Mac, but I also often have to work on Windows to test/debug. 

Is there a way to force it to use UNIX line endings automatically? If not, is there a manual way (VI command?) to do this?

Thanks!

**Comment by ThomasRedstone** [comment]
This one is pretty important.
While giving some UI to control it on a per-file basis would be handy. The vast majority of the time, people want a common behaviour project wide, so it should link into the `.editorconfig` support and ideally be supported by the local `.zed/settings.json` and other Zed config files.

**Comment by Jimmy-Z** [comment]
I guess we should give thumb ups here: https://github.com/zed-industries/zed/issues/5294

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
